src/kakarot/stack.cairo

```diff
@@ -135,101 +106,30 @@ namespace Stack {
     // @notice Swap two elements in the stack.
     // @dev stack_index_1 and stack_index_2 are 0-based, 0 is the top of the stack.
```

Comment:
you can delete this note for devs as `stack_index_1` and `stack_index_2` have been deleted